Engine/TransactionHandlers/BacktestingTransactionHandler.cs

```diff
@@ -67,5 +75,31 @@ public override void ProcessAsynchronousEvents()
             _brokerage.SimulateMarket();
             _brokerage.Scan();
         }
+
+        /// <summary>
+        /// For backtesting we will process the order ourselves
+        /// </summary>
+        /// <param name="ticket">The <see cref="OrderTicket"/> expecting to be processed</param>
+        protected override void WaitForOrderToBeProcess(OrderTicket ticket)
+        {
+            // we process the order request our selves
+            Run();
+
+            if (!ticket.OrderSet.WaitOne(0))
+            {
+                // this shouldn't happen, but just in case...
```

Comment:
perhaps describe what this means? If I'm reading it correctly, it means that the ticket's order property was not set yet. (nice usage of wait handles 👍 )